### PR TITLE
Docs to support cloud storage for Cloud exports

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/mattermost/pull/23915

### CODE REVIEW

**Review by Willyfrog — APPROVED** [review]

File: source/manage/cloud-data-export.rst [hunk 1]
```diff
@@ -44,43 +46,66 @@ You'll be prompted for a username (use your admin user), password, and for a con
...
+      Running this command creates a full export of the server, including attached files, in a unique Amazon S3 cloud storage location, and generates a pre-signed URL for the export file.
+
+      This process can take some time, so ``mmctl`` will return immediately, and the job will run in the background on the Mattermost instance until the export is fully created. If successful, the command will immediately output a job ID, like this:
+
+      .. code::
+
+         Export process job successfully created, ID: yfrr9ku5i7fjubeshs1ksrknzc
 
-   mmctl export download r3kcj8yuwbramdt714doafi3oo_export.zip
+      From your Mattermost Cloud web instance, retrieve the file download link to the export by using the Mattermost slash command ``/exportlink [job-id|zip file|latest]``. Use the ``latest`` option to automatically pull the latest export available, or specify the download link by ``job-id`` or ``zip file``.
```
Comment (nab-77) [inline source/manage/cloud-data-export.rst:108]: We need to be explicit as to where to run the slash command. Recommend we explain "hopping" over to the webapp to run the slash command and download the export.

File: source/manage/cloud-data-export.rst [hunk 2]
```diff
@@ -44,43 +46,66 @@ You'll be prompted for a username (use your admin user), password, and for a con
...
+
+         mmctl export generate-presigned-url [exportname]
+
+      Running this command creates a full export of the server, including attached files, in a unique Amazon S3 cloud storage location, and generates a pre-signed URL for the export file.
+
+      This process can take some time, so ``mmctl`` will return immediately, and the job will run in the background on the Mattermost instance until the export is fully created. If successful, the command will immediately output a job ID, like this:
+
+      .. code::
+
+         Export process job successfully created, ID: yfrr9ku5i7fjubeshs1ksrknzc
```
Comment (nab-77) [inline source/manage/cloud-data-export.rst:106]: Can we add a section similar to that on the local export tab explaining how to show export job lists and how to check when the job is ready

**Comment by JulienTant** [comment]
@cwarnermm sorry it took me so long to give an answer to your request 😊 

The command `mmctl export list`

Will output this sort result:

```
5taugpccspnybcbkd6hxycpjsr_export.zip
aapttb7q3bfd8xz8xj9wfax3gy_export.zip
gs1pd8dryib4inz1fk6erqq59e_export.zip


There are 3 exports on local instance
```

I don't know if you need this piece of in, but the filename structure if `{job_id}_export.zip`, `{job_id}` being what you get from running `mmctl export create`

I'm reviewing the rest of the documentation now!

**Comment by cwarnermm** [comment]
Thank you, @JulienTant, for your updates to this docs PR! With your updates in place, I was able to collapse the tabbed content into a single path of work for the admin to complete. This docs PR is now ready for your final review.

**Review by JulienTant — APPROVED** [review]
LGTM @cwarnermm! thank you!!

**Comment by cwarnermm** [comment]
This PR can be merged at any time.

**Pull request merged**